src/resources/Notifications.js

```diff
@@ -0,0 +1,122 @@
+import Resource from './Resource'
+
+export default class Notifications extends Resource {
+  source = 'notifications';
+
+  /**
+   * List notifications by user
+   *
+   * @param {int} userId - Filter features by userId.
+   * @param {int} start- Starting row for pagination.
+   * @param {int} limit - Limit results per request.
+   * @param {string} [type=''] - Filter results by type.
+   * @param {bool} [unread=''] - Filter results by unread.
+   * @param {bool} [noCache=false] - Flag to turn off the cache.
+   * @return {Object[]} - Notifications.
+   */
+  async list (
+    userId,
+    start,
+    limit,
+    type,
+    unread,
+    noCache = false
+  ) {
+    const params = {
+      userId,
+      start,
+      limit,
+      type,
+      unread,
+      noCache
+    }
+    clean(params)
+    return await this.request({
+      path: `notifications/${userId}`,
+      params
+    })
+  }
+
+  /**
+   * Count notifications by user
+   *
+   * @param {int} userId - Filter features by userId.
+   * @param {int} start- Starting row for pagination.
+   * @param {int} limit - Limit results per request.
+   * @param {string} [type=''] - Filter results by type.
+   * @param {bool} [unread=''] - Filter results by unread.
+   * @param {bool} [noCache=false] - Flag to turn off the cache.
+   * @return {int} - Notifications.
+   */
+  async count (
+    userId,
+    start,
+    limit,
+    type,
+    unread,
+    noCache = false
+  ) {
+    const params = {
+      userId,
+      start,
+      limit,
+      type,
+      unread,
+      noCache
+    }
+    clean(params)
+    const result = await this.request({
+      path: `notifications/count/${userId}`,
+      params
+    })
+    return Array.isArray(result) ? result[0].count : result
```

Comment:
Can we not just make sure the notifications API is consistent in sending the result.